src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -314,15 +330,45 @@ async def configure_proof_dirs(
         queue.task_done()
 
 
+def get_status_and_proof_summaries(run_dict):
+    # All possible statuses can be viewed in the man page for Litani's run.json:
+    # https://awslabs.github.io/aws-build-accumulator/#litani-man_litani-run.json
+    count_statuses = {}
+    proofs = [["Proof", "Status"]]
+    for proof_pipeline in run_dict["pipelines"]:
+        status_pretty_name = proof_pipeline["status"].title().replace("_", " ")
+        try:
+            count_statuses[status_pretty_name] += 1
+        except KeyError:
+            count_statuses[status_pretty_name] = 1
+        proof = proof_pipeline["name"]
+        proofs.append([proof, status_pretty_name])
+    statuses = [["Status", "Count"]]
+    for status, count in count_statuses.items():
+        statuses.append([status, count])
+    return [statuses, proofs]
```

Comment:
Can I suggest:
```
return [{
  "data": statuses,
  "show_index": False,
}, {
  "data": proofs,
  "show_index": True,
}]
```
see below

Reply:
To be honest, looking at the output you posted above, I'm not sure what the value of `show_index` even is. Especially since the proofs themselves aren't in any particular order.

Reply:
I perceived the value of the index being that someone looking at the proof-specific table could:
1) Point/refer to a proof at a specific index when interpreting/sharing results
2) Understand the total number of proofs

However, on second thought, I will be removing the index. The total is already available elsewhere.

Reply:
Another reason for removing the index is that rendering the table later breaks in certain platforms.